alphagov/paas-cf
# [#132745311] Datadog and syslog releases

## What
Story: [Migrate existing bosh release builds to use pullrequest-resource based builds](https://www.pivotaltracker.com/story/show/132745311)

Use CI-built datadog and syslog releases (same version as before).

## How to review
* Deploy with datadog enabled
* Check datadog
* Check logsearch

## Who can review
Not me
